kube_dns/README.md

```diff
@@ -18,13 +17,11 @@ The Kube-dns check is included in the [Datadog Agent][1] package, so you don't n
 
```

Comment:
Here we call it `Kube-dns` whereas on lines 53 and 57 it is `Kube-DNS`

Reply:
Updating in followup PR. Looks like Google calls it `kube-dns`: https://cloud.google.com/kubernetes-engine/docs/how-to/kube-dns